project/Dependencies.scala

```diff
@@ -1084,17 +1086,22 @@ object Dependencies {
   val `maven-launcher` = libraryDependencies := Seq(
     // These dependencies come from https://github.com/apache/maven/blob/master/apache-maven/pom.xml, they are
```

Comment:
Lack of sync between these dependencies here and what is on the `pom.xml` linked above was the cause of `mavenTest` failures. Thanks to @ignasi35 for pointing out how to better debug this problem.